src/data/mystery-encounters/mystery-encounter-data.ts

```diff
@@ -14,3 +16,185 @@ export class MysteryEncounterData {
     }
   }
 }
+
+export class MysteryEncounterAuras {
```

Comment:
just a note/reminder since we've already discussed. This logic can probably sit in Modifier class extensions :)